include/opw_kinematics/opw_parameters_examples.h

```diff
@@ -2,15 +2,8 @@
 #define OPW_PARAMETER_EXAMPLES_H
 
 #include "opw_kinematics/opw_kinematics.h"
+#include <opw_kinematics/opw_macros.h>
```

Comment:
Would we perhaps want to `#include` the file in the `.cpp`? It doesn't appear to be used in this file.

Reply:
Sounds good! I moved the DEPRECATED macro into the opw_macros.h file so they are all located in the same file which is why it now required in this header.

Reply:
The macro is not used in this header.

So it would seem `opw_macros.h` is not needed in this header either.

Ideally it's only included in files that actually use it, to keep preprocessing and compilation time down.